xbmc/addons/include/xbmc_pvr_types.h

```diff
@@ -255,28 +302,92 @@ extern "C" {
+                                                              filled if iPrioritiesSize > 0 */
+    int          iPrioritiesDefault;                        /*!< @brief (optional) The default value for PVR_TMER.iPriority. Must be filled if iPrioritiesSize > 0 */
+
+    /* lifetime value definitions */
+    unsigned int iLifetimesSize;                            /*!< @brief (required) Count of possible values for PVR_TMER.iLifetime. 0 means lifetime
+                                                              is not supported by this timer type or no own value definition wanted, but to use Kodi defaults
+                                                              of 1..365. */
+    PVR_TIMER_TYPE_ATTRIBUTE_INT_VALUE
+      lifetimes[PVR_ADDON_TIMERTYPE_VALUES_ARRAY_SIZE];     /*!< @brief (optional) Array containing the possible values for PVR_TMER.iLifetime. Must be
+                                                              filled if iLifetimesSize > 0 */
+    int          iLifetimesDefault;                         /*!< @brief (optional) The default value for PVR_TMER.iLifetime. Must be filled if iLifetimesSize > 0 */
+
+    /* prevent duplicate episodes value definitions */
+    unsigned int iPreventDuplicateEpisodesSize;             /*!< @brief (required) Count of possible values for PVR_TMER.iPreventDuplicateEpisodes. 0 means duplicate
+                                                              episodes prevention is not supported by this timer type or no own value definition wanted, but to use
+                                                              Kodi defaults. */
+    PVR_TIMER_TYPE_ATTRIBUTE_INT_VALUE
+      preventDuplicateEpisodes[PVR_ADDON_TIMERTYPE_VALUES_ARRAY_SIZE];
+                                                            /*!< @brief (optional) Array containing the possible values for PVR_TMER.iPreventDuplicateEpisodes.. Must
+                                                              be filled if iPreventDuplicateEpisodesSize > 0 */
+    unsigned int iPreventDuplicateEpisodesDefault;          /*!< @brief (optional) The default value for PVR_TMER.iPreventDuplicateEpisodesSize. Must be filled if iPreventDuplicateEpisodesSize > 0 */
+
+    /* recording folder list value definitions */
+    unsigned int iRecordingGroupSize;                       /*< @brief (required) Count of possible values of PVR_TIMER.iRecordingGroup. 0 means folder lists are not supported by this timer type */
+    PVR_TIMER_TYPE_ATTRIBUTE_INT_VALUE
+      recordingGroup[PVR_ADDON_TIMERTYPE_VALUES_ARRAY_SIZE];
+                                                            /*!< @brief (optional) Array containing the possible values of PVR_TMER.iRecordingGroup. Must be filled if iRecordingGroupSize > 0 */
+    unsigned int iRecordingGroupDefault;                    /*! @brief (optional) The default value for PVR_TIMER.iRecordingGroup. Must be filled in if PVR_TIMER.iRecordingGroupSize > 0 */
```

Comment:
doxy is missing `<`. not that we've ever generated docs, but we may do that some day ;-)